docs/README.md

```diff
@@ -0,0 +1,60 @@
+# OpenTelemetry Collector
+
```

Comment:
Can we mark this document with a [stability level](https://github.com/open-telemetry/opentelemetry-specification/blob/main/oteps/0232-maturity-of-otel.md#maturity-levels) to give us a chance to refine it before it is considered final?

Reply:
Not a blocker. Can be merged without it, but if we anticipate changes to the doc it would be good to have stability level listed.